xample-malware/trickbot.md

```diff
@@ -11,8 +11,9 @@ TrickBot
 ========
 Trojan spyware program that has mainly been used for targeting banking sites. TrickBot is written in the C++ programming language.
 
-Behaviors
----------
+
+ATT&CK Techniques
```

Comment:
Trickbot is in ATT&CK - are all the techniques listed new to the analysis (not in ATT&CK)?

Reply:
Ah no this was present in the file beforehand, will be changing it according to new protocol